wesnoth-lib.po

```diff
@@ -776,13 +775,15 @@ msgid ""
 "defense and movement costs are purely based on the unit’s stats for "
 "<italic>text='Fungus'</italic>."
 msgstr ""
+"Trpasličí příběhy popisují různé druhy hub. U některých druhů nemá obtížnost "
+"chůze po houbovité podlaze žádný vztah k jejímu podkladu, i když je podklad "
+"stále viditelný. Obranný bonus a náklady na pohyb v tomto terénu vycházejí "
+"čistě ze statistik jednotky pro <italic>text='Fungus'</italic>."
```

Comment:
```suggestion
"čistě ze statistik jednotky pro <italic>text='houby'</italic>."
```

Reply:
Mám za to, že tam houby patří, jen jsem při vkládání návrhu omylem nestiskl start review.

Doťukneme to?